cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -120,7 +120,8 @@ function generate_prune_whitelist_flags() {
 # besides the default ones.
 extra_prune_whitelist=
 if [ -n "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" ]; then
-  extra_prune_whitelist=( "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" )
+  # Quotes break parsing space separated values
```

Comment:
This comment might be hard to understand after some time as it hard to say what quotes it refers to.
Could you rewrite to smth like:
```
# Note: we cannot have ( "${KUBECTL_EXTRA_PRUNE_WHITELIST:-}" ) as quotes break parsing space separated values
```

Reply:
Done